shared/templates/rsyslog_logfiles_attributes_modify/ansible.template

```diff
@@ -0,0 +1,70 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+{{{ ansible_instantiate_variables(PARAMETER) }}}
+
+- name: "Set rsyslog logfile configuration facts"
```

Comment:
Could you include `{{{ rule_title }}}` as prefix for the task names, please? This is valid for other tasks too.

Example: https://github.com/ComplianceAsCode/content/blob/8f512dfd63b43df1d3e6ad63a35026b5895f8ac4/shared/macros/10-ansible.jinja#L715

Reference:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

Reply:
[7d1d759](https://github.com/ComplianceAsCode/content/pull/9789/commits/7d1d75945b832a8789227681ddee31851ba79152)